admin_ui/backend/api/outbound.py

```
@@ -0,0 +1,771 @@
+
+    if ext == ".ulaw":
+        return data
+
+    try:
+        with wave.open(io.BytesIO(data), "rb") as wavf:
+            nch = wavf.getnchannels()
+            sampwidth = wavf.getsampwidth()
+            fr = wavf.getframerate()
+            nframes = wavf.getnframes()
+            frames = wavf.readframes(nframes)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)